README.md

```diff
@@ -1,29 +1,83 @@
-# Tuples 
+# Tuples
 
+## Objectives
 
-## Learning Objectives (Students will be able to ....) 
-- Describe a tuple as being a type that groups multiple values into a single compound value.
-- Explain that values within a tuple can be of any type and do not have to be of the same type as each other.
-- Iterate over the key-value pairs in a dictionary where each item is returned as a (key, value) tuple.
-- Create a tuple and name the individual elements
+1. Describe the *tuple* data type
+1. Create tuples, name and access their values
+1. Use tuples to return multiple values from a function
+1. Iterate over the elements of a dictionary using tuples
 
-## What the Student Already Knows
-A student should already understand the concept of variables, constants, Int, String, Bool, Array, Dictionary and functions but not Tuples.
+## Understanding Tuples
 
-## Outline/ Notes
+You are already familiar with arrays, which allow storing an ordered list of values of the same type. Those can be iterated over using a for-loop, like in this example with the list of seasons:
 
-* Start with the known- students know how to iterate over arrays. Show an example of iterating over an array. 
-    * Present some real world problem such as, “If you have a dictionary where the keys represent the seasons and the values represent your favorite color associated with that season, is there an easy way to print out a list that for each sentence and color that says “In the {season} my favorite color is {color}.”
-    - If you have a better example feel free to use it, but always aim for a real world example. And you can use that example throughout the rest of your reading and/or use other new examples. 
-    * The answer is tuples and this sets the stage for writing the content that covers your learning objectives above.--- You can iterate over the key-value pairs in a dictionary with a for-in loop. Each item in the dictionary is returned as a (key, value) tuple. Explain how you can decompose the tuple’s members into temporary constants as part of the iteration.
-    
+```swift
+let seasons = ["Winter", "Spring", "Summer", "Fall"]
+
+for season in seasons {
+  print(season) // e.g. Winter
+}
+```
+
+In this exercise, we will learn about another type which allows grouping multiple, related values, the *tuple*. A tuple can contain values of any type where each value can be of a different type. However, a tuple is not considered a collection, therefore it is not possible to iterate over its values with a loop.
+
+For example, you may want to store name and age of a person in a single compound value:
+
+```swift
+let person = ("Frank", 32)
+```
+
+There are multiple ways of accessing the individual values of a tuple. Using index numbers starting with zero:
+
+```swift
+print(person.0) // Frank
+print(person.1) // 32
 ```
+
+By assigning the individual values to new variables:
+
+```swift
+let (name, age) = person
+
```

Comment:
This is great! I would also like it if you broke down every element of this line of code identifying what let, name, age and person is in a clear sentence. It's important to reinforce the proper way to structure this sentence as it gives the student more confidence to be able to explain their code, they become better googlers (if that's a thing) and challenges them to see if they REALLY know what it is that's going on with this line of code.